src/NServiceBus.Transport.SQS/NServiceBus.Transport.SQS.csproj

```diff
@@ -16,10 +15,11 @@
     <PackageReference Include="AWSSDK.S3" Version="[3.7.101.10, 3.8.0)" />
     <PackageReference Include="AWSSDK.SimpleNotificationService" Version="[3.7.100.10, 3.8.0)" />
     <PackageReference Include="AWSSDK.SQS" Version="[3.7.100.9, 3.8.0)" />
-    <PackageReference Include="NServiceBus" Version="[8.0.0, 9)" />
+    <PackageReference Include="NServiceBus" Version="[8.0.0, 9.0.0)" />
     <PackageReference Include="Fody" Version="6.6.4" PrivateAssets="All" />
     <PackageReference Include="Obsolete.Fody" Version="5.3.0" PrivateAssets="All" />
     <PackageReference Include="Particular.Packaging" Version="2.3.0" PrivateAssets="All" />
+    <PackageReference Include="System.Text.Json" Version="7.0.1" />
```

Comment:
Initially, this one was for NET472 but we had to move it because otherwise the `.Default` property is not available.